src/plugins/intel_gpu/src/graph/impls/onednn/utils.cpp

```diff
@@ -273,7 +273,16 @@ int64_t get_offset(const cldnn::layout& l, dnnl::memory::desc&& desc) {
     }
 }
 
-dnnl::memory::desc layout_to_memory_desc(cldnn::layout l, dnnl::memory::format_tag target_fmt, bool flatten, bool use_strides) {
+std::tuple<dnnl::memory::desc, dnnl::memory::desc, dnnl::memory::desc>
+get_conv_memory_descs(cldnn::layout input_layout, cldnn::layout weights_layout, cldnn::layout output_layout, dnnl::memory::format_tag target_fmt) {
```

Comment:
Do we need target_fmt parameter? Probably we always use _dnnl::memory::format_tag::undef_ in Conv, Deconv.
Similar likes weight which uses any tag, how about fix to set undef tag in input and output?
Unnecessarily many parameters increase code complexity.